.github/workflows/style.yaml

```diff
@@ -104,3 +104,18 @@ jobs:
 
       - name: make style
         run: make style
+
+  slack-on-style-failure:
+    if: github.event_name == 'push' && failure()
+    name: Post Style Failure
+    runs-on: ubuntu-latest
+    needs:
+      - check-generated-files
+      - misc-checks
+      - style-check
```

Comment:
If someone adds a new job to the workflow but forgets to update this list, we'll not have notifications for that new job. I wasn't able to find a way to do something different than `needs:` in order to make this job run last. Do you know if there's a way to maintain this list automatically?

Reply:
Not that I know of either.

Reply:
A way to achieve this would be to use GH API to get all jobs of the workflow how mentioned here https://stackoverflow.com/a/71241789/484050
And compare that with the jobs mentioned in `needs` context https://docs.github.com/en/actions/learn-github-actions/contexts#needs-context

I guess, I'll create a JIRA ticket.

Reply:
https://issues.redhat.com/browse/ROX-15200